client/src/components/MultiJurisdictionAudit/RoundManagement/RoundDataEntry.tsx

```diff
@@ -1,8 +1,90 @@
 import React from 'react'
+import { useParams } from 'react-router-dom'
+import { H5 } from '@blueprintjs/core'
+import { Field, Formik, Form, FormikProps } from 'formik'
 import H2Title from '../../Atoms/H2Title'
+import useContests from '../useContests'
+import { IRound } from '../useRoundsJurisdictionAdmin'
+import Card from '../../Atoms/SpacedCard'
+import FormField from '../../Atoms/Form/FormField'
+import FormButton from '../../Atoms/Form/FormButton'
+import { testNumber } from '../../utilities'
 
-const RoundDataEntry = () => {
-  return <H2Title>Round 1 Data Entry</H2Title>
+interface IParams {
+  electionId: string
+}
+
+interface IValues {
+  results: {
+    [contestId: string]: {
+      [choiceId: string]: string | number
+    }
+  }
+}
+
+interface IProps {
+  round: IRound
+}
+
+const RoundDataEntry = ({ round }: IProps) => {
+  const { electionId } = useParams<IParams>()
+  const [contests] = useContests(electionId)
+
+  const results = contests
+    ? contests.reduce(
+        (a, contest) => ({
+          [contest.id]: contest.choices.reduce(
+            (b, choice) => ({ [choice.id]: '' }),
+            {}
+          ),
+        }),
+        {}
+      )
+    : null
+
+  const submit = (values: IValues) => {
+    console.log(values)
+  }
+
+  if (!results) return null
+  return (
+    <Formik initialValues={{ results }} enableReinitialize onSubmit={submit}>
```

Comment:
for validation, the backend requires that you enter a result for every choice for every contest. dunno if that's the right thing to do, but it seemed like the simplest place to start. might want to require that on the frontend as well

Reply:
@jonahkagan It tests each field for being a number, which also requires them to be filled.